receiver/dummyreceiver/README.md

```diff
@@ -0,0 +1,25 @@
+# Dummy Receiver
+
+| Status      |                            |
+|-------------|----------------------------|
+| stability   | alpha: metrics             |
+| Code Owners | kyma-project/observability |
+
+Dummy Receiver is an OTel Collector receiver that generates dummy telemetry data. At the moment it only supports metrics. It is useful for testing
+the OTel Collector pipeline.
+
+## How It Works
+
+It generates dummy metrics and sends them to the OTel Collector pipeline.
+
+For generated metrics see [documentation.md](documentation.md) file.
+
+## Configuration
+
+Below is an example of the configuration:
```

Comment:
```suggestion
See the following example configuration:
```

Do we expect the reader to do something with the configuration, or just admire it?

Reply:
admire would be nice :D